samples/cpp/tutorial_code/ShapeDescriptors/findContours_demo.cpp

```diff
@@ -25,10 +25,15 @@ void thresh_callback(int, void* );
 /**
  * @function main
  */
-int main( int, char** argv )
+int main( int argc, char** argv )
 {
-  /// Load source image and convert it to gray
-  src = imread( argv[1], 1 );
+  /// Load source image
+  string imageName("../data/stuff.jpg"); // by default
```

Comment:
Change this at least to simply `stuff.jpg` and then add the check I suggested. This forces the user to place the image at the correct location.